sphinx/themes/basic/static/_compat.js

```diff
@@ -0,0 +1,133 @@
+/*
+ * _compat.js
+ * ~~~~~~~~~~
+ *
+ * Compatability shim for jQuery and underscores.js.
+ *
+ * WILL BE REMOVED IN Sphinx 6.0
```

Comment:
Added this compatability module as I realised that themes may rely on the jQuery extensions defined in the original `docutils.js`. These are though all replaceable. 

- `jQuery.urldecode` -> `decodeURIComponent` (or the `URLSearchParams` API)
- `jQuery.urlencode` -> `encodeURIComponent` (or the `URLSearchParams` API)
- `jQuery.getQueryParameters` -> the `URLSearchParams` API
- `jQuery.fn.highlightText` -> the new functions in `docutils.js`